open_city_profile/settings.py

```diff
@@ -322,12 +323,30 @@
 
 AUDIT_LOGGING_ENABLED = env.bool("AUDIT_LOGGING_ENABLED")
 AUDIT_LOG_USERNAME = env.bool("AUDIT_LOG_USERNAME")
+AUDIT_LOG_FILENAME = env("AUDIT_LOG_FILENAME")
+
+_audit_stream_handler = {
+    "level": "INFO",
+    "class": "logging.StreamHandler",
+    "stream": stdout,
+}
+_audit_rotating_file_handler = {
+    "level": "INFO",
+    "class": "logging.handlers.RotatingFileHandler",
+    "filename": AUDIT_LOG_FILENAME,
+    "maxBytes": 100000000,  # 100 MiB
+    "backupCount": 1,  # Keep one previous version
+}
```

Comment:
Are these values needed here? Why not just construct the one that is needed within the following if/else construct?

Reply:
Indeed they are not needed. I suppose I was trying to use variable names to document their purposes. And ended up just duplicating the handler name. Bwah. I'll refactor this.